### DISCUSSION TITLE
Making line graph from multiple columns data in R

### DISCUSSION THREAD

**Discussion by mchen899** [teamalgoritma/community#704] [Q&A] [answered]
Here is my data
| Country | Pop 2021 | Pop 2022 |
| --------| -------- | -------- |
| A       | xxx      | xxxx     |
| B       | xxx      | xxxx     |
| C       | xxx      | xxxx     |

I want to make line chart (filter by country) (with data from column Pop 2021-2023)
So we can see the trend

How to do that?

**Comment by IrfanChairurrachman** [comment, marked as answer]
Dear @mchen899 ,

To see the trend for various years, you can plot multiple line (each line represent the year). You can do that using `pivot_longer()` function.

```
data %>% pivot_longer(cols = c("Pop 2021", "Pop 2022"), names_to = "year", values_to = "value")
```

After that, you just need to plot the line using `geom_line()` and fill the color parameter with `year` column (so each line has their own color).

May this answer helps you.
